core/src/main/java/feign/RequestTemplate.java

```diff
@@ -95,7 +95,7 @@ static String urlEncode(Object arg) {
   }
 
   private static boolean isHttpUrl(CharSequence value) {
-    return value.length() >= 4 && value.subSequence(0, 3).equals("http".substring(0, 3));
+    return value.length() >= 4 && value.subSequence(0, 4).equals("http".substring(0, 4));
```

Comment:
change make sense,

Just wonder if you have a use case, so we could attach a test

Reply:
It is my first time to pull a request. I will submit a test case next time.